sites/authentication/authentication-en.html

```diff
@@ -0,0 +1,89 @@
+	<li><code>labelExtended</code> with a maximum lenght of 25 characters allowed including spaces, this parameter is optional but must be set along with the <code>label</code> paramter and if set, it will generate the text appearing on the button for small viewport and up (sm, md, lg and xl).</li>
+</ul>
+
+<h3>Working Examples</h3>
+<ul>
+	<li><a href="contextual-signin-en.html">Contextual Sign-in button</a></li>
+	<li><a href="contextual-signin-labelExtended-en.html">Contextual Sing-in button with extended label</a></li>
+	<li><a href="contextual-signin-nolabels-en.html">Contextual Sign without custom labels</a></li>
+</ul>
+
+<h2>Signed-off / Signed-on</h2>
+
+<p>Setting the <code>type</code> parameter with the value "signedoff" or "signedon" will initiate the Signed-off / Signed-on pattern. The buttons will appear on page load and will be located on the top right-hand side of the page, below the horizontal line.</p>
+
+<h3>Specifications</h3>
+<ul>
+	<li><strong>signedoff</strong> set as value to the <code>type</code> parameter will generate the "Register" and "Sign in" buttons to appear on the top right-hand side of the page, below the horizontal line.</li>
+	<li><strong>signedon</strong> set as value to the <code>type</code> parameter will generate the "My account" and "Sign out" buttons to appear on the top right-hand side of the page, below the horizontal line along with the wording "Signed in as [full name]" right underneath.</li>
+</ul>
+
+<h3>Working Examples</h3>
+<ul>
+	<li><a href="signedoff-en.html">Authentication - Signed Off</a></li>
+	<li><a href="signedon-en.html">Authentication - Signed On</a></li>
+</ul>
+
+<h3>Previous design patterns</h3>
+<details>
+	<summary>Previous pattern (Authentication Signed off)</summary>
+	<p>The "Register" and "Sign in" buttons were located on the right-hand side of the page, at the end of the <code>&lt;header&gt;</code> section so appearing between the breadcrumb and the <code>&lt;h1&gt;</code> element.</p>
+	<p><img src="assets/previous-signedoff-en.png" /></p>
+	<h3>Code</h3>
+	<pre><code>&lt;section id="wb-so"&gt;
+	&lt;h2 class="wb-inv"&gt;Sign-on information&lt;/h2&gt;
+	&lt;div class="container"&gt;
+		&lt;div class="row"&gt;
+			&lt;div class="col-md-12"&gt;
+				&lt;button class="btn btn-default" type="button"&gt;Register&lt;/button&gt;
+				&lt;button class="btn btn-primary" type="button"&gt;Sign in&lt;/button&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/section&gt;</code></pre>
+</details>
+
+<details>
+	<summary>Previous pattern (Authentication Signed on)</summary>
+	<p>The "My account" and "Sign out" buttons along with the wording "Signed in as [full name]" right underneath were located on the right-hand side of the page, at the end of the <code>&lt;header&gt;</code> section so appearing between the breadcrumb and the <code>&lt;h1&gt;</code> element.<img src="assets/previous-signedon-en.png" /></p>
```

Comment:
Please code the HTML as the same style as the previous details/summary element where the image are outside the paragraph.

Also:
A text alternative must be provided. And it must include all visible image of text.
```suggestion
	<p>The "My account" and "Sign out" buttons along with the wording "Signed in as [full name]" right underneath were located on the right-hand side of the page, at the end of the <code>&lt;header&gt;</code> section so appearing between the breadcrumb and the <code>&lt;h1&gt;</code> element.</p>
	<p><img src="assets/previous-signedon-en.png" alt="something that include all visible textual content"/></p>
```